resources/js/src/app/components/customer/AddressInputGroup.vue

```diff
@@ -48,15 +48,15 @@
                         <!-- First name -->
                         <div class="col-12 col-sm-4">
                             <div class="input-unit" data-model="name2" v-validate:text="areNameFieldsRequired('de', 'delivery_address')">
-                                <input type="text" name="firstName" :id="'txtFirstName' + _uid" :value="value.name2" @input="emitInputEvent('name2', $event.target.value)">
+                                <input type="text" name="firstName" :id="'txtFirstName' + _uid" :value="value.name2" @input="emitInputEvent('name2', $event.target.value)"  data-testing="address-firstname">
                                 <label :for="'txtFirstName' + _uid">{{ $translate("Ceres::Template.addressFirstName") }}<template v-if="areNameFieldsRequired('de', 'delivery_address')">*</template></label>
                             </div>
                         </div>
 
                         <!-- Last name -->
                         <div class="col-12 col-sm-4">
                             <div class="input-unit" data-model="name3" v-validate:text="areNameFieldsRequired('de', 'delivery_address')">
-                                <input type="text" name="lastName" :id="'txtLastName' + _uid" :value="value.name3" @input="emitInputEvent('name3', $event.target.value)">
+                                <input type="text" name="lastName" :id="'txtLastName' + _uid" :value="value.name3" @input="emitInputEvent('name3', $event.target.value)" data-testing="address-lastname">
```

Comment:
Hier vllt einmaliger benennen da wir ja sehr viele verschiedene layouts in der selben Datei pflegen.